### PR TITLE
Kira Yoshikage's clothes

### PR DESCRIPTION
**Pull request by oprayx77**
your favorite paraphilic murderers clothes

http://puu.sh/t0cz1/157796a1b6.png

http://puu.sh/t0cxj/91a3f42730.jpg

https://puu.sh/sIIpN/30a016ac6f.jpg
it's in the threatrevend, the tie attaches to suits,
without it you can see the person's undershirt

OK LETS BE CLEAR: the tie and suit are seperate items. The onground object item has the lil tie icon next to it, when it's on your person, it looks proper. You can wear the suit with or without.
https://puu.sh/sIIpN/30a016ac6f.jpg this is the suit with the tie on.

### CODE REVIEW

**Comment by Kierany9** [comment]
>tie is off-center

holy fuck why

**Comment by oprayx77** [comment]
loomk at all those fuckin commits lol

**Comment by Spacedong** [comment]
>this many commits for a fucking sprite

**Comment by oprayx77** [comment]
dude you WANT this cock or somethin eh?

**Comment by Spacedong** [comment]
ya

**Comment by yobob591** [comment]
is it found in the autorobe or the public vendor

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -949,7 +949,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/under/jester = 1, /obj/item/clothing/head/jester = 1,
 					/obj/item/clothing/suit/hooded/carp_costume = 1,
 					/obj/item/clothing/suit/hooded/ian_costume = 1,
-					/obj/item/clothing/under/dio = 1, /obj/item/clothing/gloves/color/dio = 1, /obj/item/clothing/head/dio = 1, /obj/item/clothing/suit/dio = 1, /obj/item/clothing/shoes/dio = 1)
+					/obj/item/clothing/under/dio = 1, /obj/item/clothing/gloves/color/dio = 1, /obj/item/clothing/head/dio = 1, /obj/item/clothing/suit/dio = 1, /obj/item/clothing/shoes/dio = 1, /obj/item/clothing/under/suit_jacket/kira = 1, /obj/item/clothing/tie/kira = 1)
```
Comment (JohnGinnane) [inline code/game/machinery/vending.dm:952]: Shouldn't we also add this to the restock item too?

**Comment by ArcLumin** [comment]
Code looks good. If this gets merged before stand arrow we can pop the kira clothes in the arrow room

**Comment by ghost** [comment]
啊...啊啊啊~
when I was a kid, I saw Mona Lisa from my garmmer school art book...
the first time I saw her, with hands on her knee....
how do I say this....
I had a boner....
+1

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (oprayx77) [reply code/game/machinery/vending.dm:952]: it's a silly one-off item and im sure no one is going to need two suits, if i get around to doing a pallette swap of it in white for kosaku kawajiri i might kek

**Comment by Hackole** [comment]
more jojo can never be wrong +1

**Comment by JohnGinnane** [comment]
Merging so

**Comment by ArcLumin** [comment]
dude why did these clothes not have 100 bomb resist smh tbh fam

**Pull request merged**